src/base/async/int.c

```diff
@@ -277,15 +279,20 @@ int register_plugin_call(int num, void (*call)(struct vm86_regs *))
 
 static void emufs_helper(void)
 {
-    switch (LO(bx)) {
-    case DOS_SUBHELPER_EMUFS_REDIRECT:
-	NOCARRY;
-	break;
+  switch (LO(bx)) {
+    case DOS_SUBHELPER_EMUFS_LOADED:
```

Comment:
I don't understand...
emufs does nothing?
Looks like some unchangelogged, secret
change. Anywhere in the log why emufs.sys
should do nothing?

Reply:
A later commit where I moved the emufs_loaded variable into int.c and changed the subhelper used to check for loading.

Reply:
I mean, together with this:
`I feel like DOS_SUBHELPER_MFS_EMUFS_INIT subhelper needs to go away`
it seems like it will do nothing.
Or what is the intention?

Reply:
In either case, I think the picture will
change if you move the init part to int.c,
so lets see what will it be.

Reply:
In which case it must be clearly described
in the log, as currently I am a bit puzzled.
Its difficult to follow all the changes at once
in the head, so the important changes should
be written as text.

Reply:
At the moment EMUFS subhelper only serves to prevent emufs.sys being loaded twice, if you are okay with a user loading it twice then the subhelper can go away. In fact I'd prefer it if emufs.sys were able to detect it was already loaded itself, but I don't know if that's possible?